packages/gatsby-cli/src/structured-errors/types.ts

```diff
@@ -47,17 +47,59 @@ export interface IOptionalGraphQLInfoContext {
   plugin?: string
 }
 
-export enum Level {
+export const enum Level {
```

Comment:
Compile the enums to values: https://www.typescriptlang.org/docs/handbook/enums.html#const-enums